src/MetaModels/ItemList.php

```diff
@@ -575,6 +575,10 @@ public function prepare()
 
         $calculator = $this->paginationLimitCalculator;
         $calculator->setTotalAmount($intTotal);
+        $curPage = (int) \Input::get('page');
+        if(is_int($curPage) && $curPage > 1){
```

Comment:
This check is obsolete as you are casting to int here. Therefore the condition is always true.